Runtime/Data/Collection/ObservableList.cs

```diff
@@ -386,7 +386,7 @@ public virtual void RemoveAtCurrentIndex()
         /// </summary>
         /// <param name="removeFromFront">Whether to start the removal from the start or the end of the collection.</param>
         [RequiresBehaviourState]
-        public virtual void Clear(bool removeFromFront)
+        public virtual void Clear(bool removeFromFront = false)
```

Comment:
Shouldn't it be `true` instead? That saves performance and is in line with what `List<T>.Clear` does.

Reply:
Yeah, but then on that I would say the variable should be renamed so the default is false and the internal check is positive

so like `bool reverseCollection = false` and then

```
if(reverseCollection) {
  reverse;
}
```

the param text could read

`Whether to reverse the collection when clearing so the removal goes from end to start.`